components/03-modules/form/_form.scss

```diff
@@ -0,0 +1,51 @@
+.form {
+    &--compact {
+        @include mq($screen-s) {
+            display: flex;
+            justify-content: center;
+        }
+
+        .form__heading {
+            line-height: $button-height;
+            margin-right: 20px;
+        }
+
+        .form__content {
+            display: flex;
+        }
+
+        .input {
+            margin-bottom: 0;
+            width: 100%;
+            display: flex;
+            flex-wrap: nowrap;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .input__field {
+            width: 100%;
+            border: $button-icon-border;
+            line-height: $button-height;
+            height: $button-height;
+            border-radius: 0;
+            @include mq($screen-s) {
+                width: 290px;
+            }
+        }
+
+        .input__label {
+            margin: 0 10px 0 0;
+        }
+
+        .button {
+            margin: 0 0 0 (-$button-icon-border-width);
+        }
+    }
+
+    div.mage-error[generated] {
```

Comment:
This one should stay in theme, it's heavily Magento related, not useful in components at all.